pkg/server/router/manifest.go

```diff
@@ -186,58 +185,58 @@ func (mr ManifestRouter) DeleteManifest(ctx context.Context, w http.ResponseWrit
 	return framework.Respond(ctx, w, nil, http.StatusOK)
 }
 
-// Application
-type CreateApplicationRequest struct {
+type SubmitApplicationRequest struct {
 	ManifestID             string                              `json:"manifestId" validate:"required"`
 	PresentationSubmission exchangensdk.PresentationSubmission `json:"presentationSubmission" validate:"required"`
 }
 
-func (c CreateApplicationRequest) ToServiceRequest() manifest.CreateApplicationRequest {
-	return manifest.CreateApplicationRequest{
+func (c SubmitApplicationRequest) ToServiceRequest() manifest.SubmitApplicationRequest {
```

Comment:
```suggestion
func (sar SubmitApplicationRequest) ToServiceRequest() manifest.SubmitApplicationRequest {
```
c no longer works here, maybe `sar` or `s`?